comscore/src/main/java/com/bitmovin/player/integration/comscore/ComScoreConfiguration.java

```diff
@@ -23,4 +29,11 @@ public String getApplicationName() {
         return applicationName;
     }
 
-}
\ No newline at end of file
+    public ComScoreUserConsent getUserConsent() {
```

Comment:
please add a comment to this public method

Reply:
these are just getters and setters on the conifg. shall i comment all of them?